docs/source/gbd2017_models/causes/neoplasms/cervical_cancer/cervical_cancer_cause_model.rst

```diff
@@ -160,21 +168,33 @@ State and Transition Data Tables
      - Measure
      - Value
      - Notes
-   * - S
+   * - Susceptible
      - prevalence
-     - 1 - (prev_BCC + prev_c432)
+     - 1 - (prev_hrHPV + prev_BCC + prev_c432)
      - derived, used only at initialization
-   * - S
+   * - Susceptible
      - excess mortality rate
      - 0
-     - No EMR for S state
-   * - S
+     - No EMR for susceptible state
+   * - Susceptible
      - disabilty weights
      - 0
-     - No disability weights for S state
+     - No disability weights for susceptible state
+   * - hrHPV-infected
+     - prevalence
+     - 19.0% (95%CI, 17.1-20.9)
```

Comment:
We will need to include an age-pattern for this, and if I have understood correctly, there is not an evidence base of incidence data to draw on.  Let's include information on how we proceed in this document.